base/MAPL_Generic.F90

```diff
@@ -5905,7 +5905,7 @@ subroutine MAPL_ESMFStateReadFromFile(STATE,CLOCK,FILENAME,MPL,HDR,RC)
              call ESMF_AttributeGet(MPL%GRID%ESMFGRID,'GridType',value=grid_type,rc=status)
              _VERIFY(status)
           end if
-          if (trim(grid_type) /= 'Tripolar' .and. trim(grid_type) /= 'llc' .and. trim(grid_type) /= 'External') then
+          if (trim(grid_type) == 'Cubed-Sphere') then
```

Comment:
I think a comment here is probably important.    Just so the next person does not come along and say "Hey we use a factory everywhere this should work for lat-lon too.".    Just some sort of cautionary note.

Reply:
I added a comment